scripts/milestone.py

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import sys
+
+import requests
+
+# from collections import OrderedDict
```

Comment:
Is this PR work in progress? Otherwise all the commented code should be removed.

Reply:
Yep I didn't have access to add the label myself. This will all be cleaned up eventually!